src/coprocessor/metrics.rs

```diff
@@ -67,6 +68,26 @@ lazy_static! {
         "Total bytes of response body"
     )
     .unwrap();
+    pub static ref COPR_ACQUIRE_SEMAPHORE_RESULT: CoprAcquireSemaphoreResultCounterVec =
+        register_static_int_counter_vec!(
+            CoprAcquireSemaphoreResultCounterVec,
+            "tikv_coprocessor_acquire_semaphore_result",
```

Comment:
Could give it a name like `tikv_coprocessor_acquire_semaphore_type`.

Reply:
I hesitated which to use...If you like I am willing to use `_type`.